modules/flowable-spring-boot/flowable-spring-boot-samples/flowable-spring-boot-sample-rest-api-security/src/test/java/org/flowable/test/spring/boot/SecurityAutoConfigurationTest.java

```diff
@@ -20,6 +21,7 @@
 /**
  * @author Josh Long
  */
+@Ignore("Running this and the RestApiSecurityApplicationTest does not work. Running them separately it is fine")
```

Comment:
For some reason having this and the `RestApiSecurityApplicationTest` does not work. It complains in duplicate keys